### PR TITLE
Add automated builds for windows/linux + fix debian version/upload

### LINKED ISSUE #144: Releases on github should contain binaries
Although we currently build binaries for linux and windows, it looks like these are not added to the releases, at least not for version 4.2.3.

### LINKED ISSUE #148: Adjust debian package name
Still cn-ws + not yet published

### PR DESCRIPTION
**Pull request by johanvdw**
Binaries will now be attached to releases (closes #144):
image

fixes debian workflow which closes #148, #131

Lastly, most build actions are now done on github servers, except the last step to publish on our apt repository. By publishing on github, we can adjust the workflow for vpo so they no longer need our repository for this. For this, the docker containers were rebuilt and added to ghcr.io.

### CODE REVIEW

File: .github/workflows/publish_packages.yml [hunk 1]
```diff
@@ -14,7 +14,28 @@ env:
...
+    - uses: actions/download-artifact@master
+      with:
+        name: linux-amd64-binary
+        path: linux
+    - uses: actions/download-artifact@master
+      with:
+        name: windows-compiled
+        path: windows
+    - uses: ncipollo/release-action@v1
+      with:
```
Comment (johanvdw) [inline .github/workflows/publish_packages.yml:35]: we could consider adding prerelease: true below, to make sure a release is only an actual release after human review.

**Review by daarende — APPROVED** [review]
As far as I can judge, this seems ok

**Comment by Sachagobeyn** [comment]
@johanvdw : one question: is the build of the binary checked in any way, i.e. does the windows / linux binary run? Or is the unit testing sufficient to check this (i don't know how the flow is)

**Comment by johanvdw** [comment]
> @johanvdw : one question: is the build of the binary checked in any way, i.e. does the windows / linux binary run? Or is the unit testing sufficient to check this (i don't know how the flow is). The build is also failing

The linux binary is used in the unit tests. The windows binary is not yet tested.

**Pull request merged**